src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -8039,11 +8082,8 @@ bool FEQX(Real64 const a, // values to compare, fractional tolerance
     Real64 d;
     Real64 tolAbsX;
 
-    if (present(tolAbs)) {
```

Comment:
Another case where the optional is being used to set a default value, so use the C++ feature.